exasol_sphinx_github_pages_generator/deploy_github_pages.py

```diff
@@ -72,16 +65,17 @@ def checkout_target_branch_as_worktree(target_branch, worktree, push_origin):
             run(["git", "reset", "--hard", gh_pages_root_branch])
             print(f"Delete root branch {gh_pages_root_branch}")
             run(["git", "branch", "-D", gh_pages_root_branch])
-        # popd ?
-
+        os.chdir(currentworkdir)
 
-def build_and_copy_documentation(build_dir, worktree, source_branch, source_dir):
+def build_and_copy_documentation(build_dir, worktree, source_branch, source_dir, module_path):
     print("Build with sphinx")
-    run(["sphinx-build", "-M", "html", source_dir, build_dir, "-W"])
+    currentworkdir = os.getcwd()
+    print("currentworkdir :" + currentworkdir)
+    run(["sphinx-apidoc", "-T", "-d", "1", "-e", "-o", "api", module_path])
```

Comment:
Would be good to add some documentation what this command actually does.

Reply:
done